ember-toucan-core/src/components/form/controls/textarea.hbs

```diff
@@ -1,6 +1,7 @@
 <textarea
-  class="min-h-6 bg-overlay-1 focus:outline-none shadow-focusable-outline focus:shadow-focus-outline block h-20 rounded-sm p-1
-    {{if @isDisabled 'text-disabled' 'text-titles-and-attributes'}}"
+  class="min-h-6 bg-overlay-1 focus:outline-none focus:shadow-focus-outline block h-20 rounded-sm p-1
+    {{if @isDisabled 'text-disabled' 'text-titles-and-attributes'}}
+    {{if @hasError 'shadow-error-outline' 'shadow-focusable-outline'}}"
```

Comment:
With CSS classes, we can't just tack this on in `textarea-field` as the error and focus shadows clobber each other and one "wins".  Instead it made it easier to conditionally apply this at the control level.